From 475ce2af8c9c5c96ee903899d6b268108252c69b Mon Sep 17 00:00:00 2001
From: Debasish Biswas <debasishbsws.dev@gmail.com>
Date: Tue, 18 Feb 2025 23:32:27 +0800
Subject: [PATCH] feat: Remove modules from bundled gems list to avoid unnecessary duplication.

Removed `rexml` and `net-imap` from `gems/bundled_gems` as they are already provided as separate packages in Wolfi

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>
---
 gems/bundled_gems | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gems/bundled_gems b/gems/bundled_gems
index 3873767596..d4f50fac6b 100644
--- a/gems/bundled_gems
+++ b/gems/bundled_gems
@@ -10,10 +10,8 @@ minitest            5.25.4  https://github.com/minitest/minitest
 power_assert        2.0.5   https://github.com/ruby/power_assert
 rake                13.2.1  https://github.com/ruby/rake
 test-unit           3.6.7   https://github.com/test-unit/test-unit
-rexml               3.4.0   https://github.com/ruby/rexml
 rss                 0.3.1   https://github.com/ruby/rss
 net-ftp             0.3.8   https://github.com/ruby/net-ftp
-net-imap            0.5.6   https://github.com/ruby/net-imap
 net-pop             0.1.2   https://github.com/ruby/net-pop
 net-smtp            0.5.1   https://github.com/ruby/net-smtp
 matrix              0.4.2   https://github.com/ruby/matrix
-- 
2.39.5 (Apple Git-154)

